src/java.base/share/classes/sun/launcher/resources/launcher_de.properties

```diff
@@ -24,22 +24,25 @@
 
 # Translators please note do not translate the options themselves
 java.launcher.X.macosx.usage=\nDie folgenden Optionen sind für macOS spezifisch:\n    -XstartOnFirstThread\n                      Führt die main()-Methode für den ersten (AppKit-)Thread aus\n    -Xdock:name=<Anwendungsname>\n                      Setzt den im Dock angezeigten Standardanwendungsnamen außer Kraft\n    -Xdock:icon=<Pfad zu Symboldatei>\n                      Setzt das im Dock angezeigte Standardsymbol außer Kraft\n\n
 
+# Translators please note do not translate the options themselves
+java.launcher.opt.concise.header  =   Verwendung: java [Java-Optionen...] <Anwendung> [Anwendungsargumente...]\n\nDabei ist <Anwendung> einer der folgenden Werte:\n  <Hauptklasse>                Zum Ausführen der Hauptmethode einer kompilierten Hauptklasse\n  -jar <JAR-Datei>.jar         Zum Ausführen der Hauptklasse eines JAR-Archivs\n  -m <Modul>[/<Hauptklasse>] Zum Ausführen der Hauptklasse eines Moduls\n  <Quelldatei>.java          Zum Kompilieren und Ausführen eines Quelldateiprogramms\n\nDabei sind die folgenden wichtigen Java-Optionen verfügbar:\n  --class-path <Classpath>\n    <Classpath> ist eine durch "{0}" getrennte Liste der Verzeichnisse und JAR-Archive, in denen nach Klassendateien gesucht werden soll\n  --module-path <Modulpfad>\n    <Modulpfad> ist eine durch "{0}" getrennte Liste der Verzeichnisse und JAR-Archive, in denen nach Modulen gesucht werden soll\n  -version\n    Zum Ausgeben der Produktversion in den Fehlerstream und Beenden des Vorgangs\n\nFür weitere Verwendungshilfe:           java --help\nFür eine interaktive Java-Umgebung:    jshell
```

Comment:
This gives the following output:
```
Verwendung: java [Java-Optionen...] <Anwendung> [Anwendungsargumente...]

Dabei ist <Anwendung> einer der folgenden Werte:
  <Hauptklasse>                Zum Ausführen der Hauptmethode einer kompilierten Hauptklasse
  -jar <JAR-Datei>.jar         Zum Ausführen der Hauptklasse eines JAR-Archivs
  -m <Modul>[/<Hauptklasse>] Zum Ausführen der Hauptklasse eines Moduls
  <Quelldatei>.java          Zum Kompilieren und Ausführen eines Quelldateiprogramms

Dabei sind die folgenden wichtigen Java-Optionen verfügbar:
  --class-path <Classpath>
    <Classpath> ist eine durch ":" getrennte Liste der Verzeichnisse und JAR-Archive, in denen nach Klassendateien gesucht werden soll
  --module-path <Modulpfad>
    <Modulpfad> ist eine durch ":" getrennte Liste der Verzeichnisse und JAR-Archive, in denen nach Modulen gesucht werden soll
  -version
    Zum Ausgeben der Produktversion in den Fehlerstream und Beenden des Vorgangs

Für weitere Verwendungshilfe:           java --help
Für eine interaktive Java-Umgebung:    jshell
```

It would be nicer if the `Zum ...` texts would be aligned, and the `java` and `jshell` at the end would be aligned. I.e.:

```
Verwendung: java [Java-Optionen...] <Anwendung> [Anwendungsargumente...]

Dabei ist <Anwendung> einer der folgenden Werte:
  <Hauptklasse>                Zum Ausführen der Hauptmethode einer kompilierten Hauptklasse
  -jar <JAR-Datei>.jar         Zum Ausführen der Hauptklasse eines JAR-Archivs
  -m <Modul>[/<Hauptklasse>]   Zum Ausführen der Hauptklasse eines Moduls
  <Quelldatei>.java            Zum Kompilieren und Ausführen eines Quelldateiprogramms

Dabei sind die folgenden wichtigen Java-Optionen verfügbar:
  --class-path <Classpath>
    <Classpath> ist eine durch ":" getrennte Liste der Verzeichnisse und JAR-Archive, in denen nach Klassendateien gesucht werden soll
  --module-path <Modulpfad>
    <Modulpfad> ist eine durch ":" getrennte Liste der Verzeichnisse und JAR-Archive, in denen nach Modulen gesucht werden soll
  -version
    Zum Ausgeben der Produktversion in den Fehlerstream und Beenden des Vorgangs

Für weitere Verwendungshilfe:          java --help
Für eine interaktive Java-Umgebung:    jshell
```